dashboard/README.md

```diff
@@ -62,8 +37,7 @@ Now, to access the React app, simply run `kubeapps dashboard` as you usually
 would:
 
 ```
-cd ../kubeapps
-./kubeapps dashboard
+kubeapps dashboard --port=5000
```

Comment:
why port 5000? wouldn't this be port 3000?

Reply:
3000 is the port the local node server runs on, 5000 is the port I've been using for the Dashboard proxy (whether I'm using telepresence or not to access it). No real reason, the random port is often a pain during development so I just picked one.